README.md

```diff
@@ -1,15 +1,14 @@
-# global-ray-intellij
+# A PhpStorm plugin that provides autocompletion for Global Ray
 
 ![Build](https://github.com/olivernybroe/global-ray-intellij/workflows/Build/badge.svg)
 [![Version](https://img.shields.io/jetbrains/plugin/v/18749.svg)](https://plugins.jetbrains.com/plugin/18749)
 [![Downloads](https://img.shields.io/jetbrains/plugin/d/18749.svg)](https://plugins.jetbrains.com/plugin/18749)
 
-<!-- Plugin description -->
```

Comment:
We need to keep this part. 

I use them for defining what text from the readme get's put into the description part on intellij marketplace

Reply:
Fixed!